XcodeServerSDK/Utils/ContainerExtensions.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ContainerExtensions.swift
+//  XcodeServerSDK
+//
+//  Created by Honza Dvorsky on 21/06/2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+extension SequenceType {
+    
+    func mapThrows<T>(@noescape transform: (Self.Generator.Element) throws -> T) throws -> [T] {
```

Comment:
Yup, I had no idea that existed. `rethrows` is actually really smart, it only marks the function as `throws` if the closure also `throws`. So you don't *always* need the `try` keyword when calling it.

Now that I think about it, this is exactly how they should change the default implementations of `map` etc - to use `rethrows` so that it supports throwing closures.

Anyway, thanks @esttorhe!